addons/web/static/src/core/emoji_picker/emoji_picker.js

```diff
@@ -306,6 +312,9 @@ export class EmojiPicker extends Component {
     }
 
     getEmojis() {
+        if (this.recentEmojis.length > 0 && this.searchTerm.length > 1) {
```

Comment:
Why is `this.searchTerm.length > 1` used? Shouldn't it be greater than 0 instead?